### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Added Home Page in trendy-t and divide features into Components.
List of Components.
1.  HomeScreen
2. Category
3. Offer 
4. Product 
5. Services
Screenshot (218)
Screenshot (217)
Home Page (2)

### CODE REVIEW

File: src/Pages/Home-Page/Home.jsx [hunk 1]
```diff
@@ -4,12 +4,24 @@ import "./responsive.css";
 import { Nav } from "../../components/Nav/Nav";
 import { NavForMobile } from "../../components/NavForMobile/NavForMobile.js";
 import { Footer } from "../../components/Footer/Footer";
+import { HomeScreen } from "./Components/HomeScreen";
+import { Category } from "./Components/Category";
+import { Offer } from "./Components/Offer";
+import { Product } from "./Components/Product";
+import { Services } from "./Components/Services";
```
Comment (Bhakti199) [inline src/Pages/Home-Page/Home.jsx:11]: you can create a common file `index` here in components and export components from it, will be easy to access.

File: src/Pages/Home-Page/Home.jsx [hunk 2]
```diff
@@ -4,12 +4,24 @@ import "./responsive.css";
...
     <div className="home-page">
       <Nav />
       <NavForMobile />
+      <main>
+        <HomeScreen />
+        <Category />
+        <Offer />
+        <Product />
+        <Services />
+      </main>
```
Comment (Bhakti199) [inline src/Pages/Home-Page/Home.jsx:24]: This is good, dividing into small components.

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me, proper folder structure is maintained.
This PR is good to merge.

**Review by neetukumari4858 — APPROVED** [review]
Overall the code is well structured.it is good to merge

File: src/Pages/Home-Page/Home.jsx (hunk 1 above)
Comment (SaurabhChawda) [reply src/Pages/Home-Page/Home.jsx:11]: Okay, I will Create Index File and Shift it.

File: src/Pages/Home-Page/Home.jsx (hunk 2 above)
Comment (SaurabhChawda) [reply src/Pages/Home-Page/Home.jsx:24]: ThankYou

**Comment by SaurabhChawda** [comment]
> Overall the code is well structured.it is good to merge

ThankYou

**Pull request merged**